interop/test_utils.go

```diff
@@ -933,3 +978,106 @@ func (s *testServer) HalfDuplexCall(stream testgrpc.TestService_HalfDuplexCallSe
 	}
 	return nil
 }
+
+// DoORCAPerRPCTest performs a unary RPC that enables ORCA per-call reporting
+// and verifies the load report sent back to the LB policy's Done callback.
+func DoORCAPerRPCTest(tc testgrpc.TestServiceClient) {
```

Comment:
Im assuming the values in the report are pre-determined in the test spec?

Reply:
it might be useful to mention the [spec](https://github.com/grpc/grpc/blob/e97f632886918a5cc603e8a510dd2d6ee4858e01/doc/interop-test-descriptions.md#orca_per_rpc) somewhere in the doc, if not already.